console/web3ext/web3ext.go

```diff
@@ -1012,6 +1012,11 @@ web3._extend({
 			call: 'subbridge_unsubscribeBridge',
 			params: 2
 		}),
+		new web3._extend.Method({
+			name: 'KASAnchor',
+			call: 'subbridge_kASAnchor',
```

Comment:
```suggestion
			call: 'subbridge_kasAnchor',
```

Reply:
`subbridge_kASAnchor` looks werid. But this is for `subbridge.KASAnchor()`. :)